llm/server.go

```diff
@@ -667,7 +662,7 @@ type completion struct {
 
 type CompletionRequest struct {
 	Prompt  string
-	Format  string
+	Format  json.RawMessage
```

Comment:
Would it be worth us making this a grammar? The `server` package could take care of generating that and this layer would only think in grammars. This can be a future change – in which case we should add a TODO or elaborate on the one below

Reply:
On this level IMO it makes sense to leave this as format and server.go or sampling packages can take care of how we make use of that format